remoteappmanager/static/js/home/controller.js

```diff
@@ -51,5 +51,7 @@ require([
         ga_observer.trigger_application_starting(application.app_data.image.name);
     });
 
+    app_list_view.$on('entry_clicked', function() { app_view.focus_iframe(); });
```

Comment:
As I said in the other comment, it sounds like "clicked" or "click" are not built-in events for the `$on` method of Vue. I think it's not really related to the javascript method `addEventListener("click", myFunction)`. We can only use the `$on`method with custom events and trigger them with the `$emit` method.